api/api.proto

```diff
@@ -2483,3 +2483,10 @@ message CatalogResponse {
 	// Report of total directories and files count
 	Report report = 2;
 }
+
+message LocateResponse {
```

Comment:
Could you add documentation to the `LocateResponse` object? What it is used for, why it is needed, etc.